docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-customer-specific-prices-feature-integration.md

```diff
@@ -0,0 +1,198 @@
+template: feature-integration-guide-template
+---
+
+This document describes how to integrate the Merchant Portal - Marketplace Customer Specific Prices feature into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Merchant Portal - Marketplace Customer Specific Prices feature core.
+
+### 1) Install the required modules using Composer
+
+Install the required module:
+
+```bash
+composer require spryker/price-product-merchant-relationship-merchant-portal-gui:"^1.0.0" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following modules were installed:
+
+| MODULE    | EXPECTED DIRECTORY        |
+|-----------|---------------------------|
+| PriceProductMerchantRelationshipMerchantPortalGui | vendor/spryker/price-product-merchant-relationship-merchant-portal-gui |
+
+{% endinfo_block %}
+
+
+### 2) Set up database schema
+
+```bash
+console propel:install
+```
+
+{% info_block warningBox "Verification" %}
+
+Verify that the following changes have been applied by checking your database:
+
+| DATABASE ENTITY | TYPE   | EVENT     |
+|-----------------|--------|-----------|
+| spy_acl_rule.bundle   | column | updated   |
+| spy_acl_rule.controller   | column | updated   |
+
+
+{% endinfo_block %}
+
+### 3) Set up the transfer objects
+
+Generate transfer changes:
+
+```bash
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that the following changes were applied in transfer objects:
+
+| TRANSFER  | TYPE  | EVENT | PATH |
+| - | - | - | - |
+| PriceProductCollectionDeleteCriteria | class | Created | src/Generated/Shared/Transfer/PriceProductCollectionDeleteCriteriaTransfer |
+| PriceProductCollectionResponse | class | Created | src/Generated/Shared/Transfer/PriceProductCollectionResponseTransfer |
+| PriceProductMerchantRelationshipCollectionDeleteCriteria | class | Created | src/Generated/Shared/Transfer/PriceProductMerchantRelationshipCollectionDeleteCriteriaTransfer |
+| PriceProductMerchantRelationshipCollectionResponse | class | Created | src/Generated/Shared/Transfer/PriceProductMerchantRelationshipCollectionResponseTransfer |
+| PriceProductTableView | class | Created | src/Generated/Shared/Transfer/PriceProductTableViewTransfer |
+| PriceProductTableCriteria | class | Created | src/Generated/Shared/Transfer/PriceProductTableCriteriaTransfer |
+| PriceProductCriteria.withAllMerchantPrices | property | Created | src/Generated/Shared/Transfer/PriceProductCriteriaTransfer |
+| PriceProductDimension.idPriceProductDefault | property | Created | src/Generated/Shared/Transfer/PriceProductDimensionTransfer |
+| PriceProduct.priceTypeName | property | Created | src/Generated/Shared/Transfer/PriceProductTransfer |
+| PriceProductDimension.name | property | Created | src/Generated/Shared/Transfer/PriceProductDimensionTransfer |
+
+{% endinfo_block %}
+
+### 4) Configure price product group key pre-build plugins
```

Comment:
All the plugins sections we can grouped in "Set up behavior" section with the plugins table accordingly.
Example: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-order-management-feature-integration.md